include/wx/generic/busyinfo.h

```diff
@@ -37,12 +37,15 @@ class WXDLLIMPEXP_CORE wxBusyInfo : public wxObject
         Init(wxBusyInfoFlags().Parent(parent).Label(message));
     }
 
+    void UpdateText(const wxString& str, bool is_markup = true);
```

Comment:
Sorry, I don't think bool `is_markup` makes for a good API.

We might just take `wxBusyInfoFlags` and use either its `Text()` or `Label()`, depending on which one is not empty.

Or, maybe simpler, we could have `UpdateText()` and `UpdateLabel()`, with the understanding that the former takes markup and the latter uses plain text only.

Reply:
Yeah, two functions seems to me even better. Fixed.